automation/run-tests.sh

```diff
@@ -54,8 +54,8 @@ function add_extra_networks {
 
 function dump_network_info {
     docker_exec '
-      nmcli dev; \
-      nmcli con; \
+      env PAGER="" nmcli dev; \
```

Comment:
I guess it works but AFAIK env is not needed here. `PAGER= nmcli dev` should work as good (Also it seems that the dev command does not use the pager at all but it makes sense for the connection command.